### DISCUSSION TITLE
Support for ecdh keys using curve secp256k1 is disabled

### DISCUSSION THREAD

**Discussion by hp8wvvvgnj6asjm7** [openpgpjs/openpgpjs#1451] [Q&A]
How to enable it?

**Comment by larabr** [comment]
You need to change the `rejectCurves` option in `config`, which by default also includes `secp256k1` (see https://github.com/openpgpjs/openpgpjs/pull/1395).
e.g. when decrypting a message, you can do
```js
... = await openpgp.decrypt({
  message,
  decryptionKeys,
  config: { rejectCurves: new Set() } // overwrite default configuration for rejectCurves 
)
```
